extensions/blocks/contact-info/edit.native.js

```diff
@@ -7,33 +7,23 @@ import { View } from 'react-native';
 /**
  * Internal dependencies
  */
+import styles from './editor.scss';
+
 const ALLOWED_BLOCKS = [
 	'jetpack/address',
 	'jetpack/email',
 	'jetpack/phone',
-	'core/paragraph',
-	'core/image',
-	'core/heading',
-	'core/gallery',
-	'core/list',
-	'core/quote',
-	'core/shortcode',
-	'core/audio',
-	'core/code',
-	'core/cover',
-	'core/html',
-	'core/separator',
-	'core/spacer',
-	'core/subhead',
-	'core/video',
+  'core/heading',
```

Comment:
Could you switch from spaces to tabs, to stay consistent with the rest of the files?